src/facade/facade_test.cc

```diff
@@ -6,19 +6,34 @@
 
 #include <absl/strings/match.h>
 
+#include "base/logging.h"
+
 namespace facade {
 
 using namespace testing;
 using namespace std;
 
 bool RespMatcher::MatchAndExplain(const RespExpr& e, MatchResultListener* listener) const {
```

Comment:
do we have to use const & for const RespExpr& e ?
if not we can skip the copy of line 17

Reply:
Good idea! Done :)